app/config.local.js

```diff
@@ -29,3 +34,4 @@ kubernetes: "dev-mode"
 };
 
 window.MOCK_ORIGIN_SERVICES = false;
+window.MOCK_SERVICES_DATA = true;
```

Comment:
...although, I'm wondering if variables like 'window.MOCK_SERVICES_DATA' should just be set in the DataManager class.  I don't think it needs to be user modifiable in config.local.js.